### PR TITLE
[hi] Localize application-architect.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/application-architect.md` to Hindi `content/hi/docs/reference/glossary/application-architect.md`

Fixes :- #35944

### CODE REVIEW

**Comment by bishal7679** [comment]
@divya-mohan0209 @anubha-v-ardhan 
Could you please leave some suggestion for this PR
It would be really helpful

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by bishal7679** [comment]
/assign @mittalyashu

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/docs/reference/glossary/application-architect.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: एप्लीकेशन आर्किटेक्ट
+id: application-architect
+date: 2018-04-12
+full_link:
+short_description: >
+  किसी एप्लिकेशन के उच्च-स्तरीय डिज़ाइन के लिए जिम्मेदार व्यक्ति।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/application-architect.md:7]: Wouldn't रचना be better than the literal translation of design?

File: content/hi/docs/reference/glossary/application-architect.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+aka:
+tags:
+  - user-type
+---
+
+किसी एप्लिकेशन के उच्च-स्तरीय डिज़ाइन के लिए जिम्मेदार व्यक्ति।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/application-architect.md:14]: Same as above.

File: content/hi/docs/reference/glossary/application-architect.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+<!--more-->
+
+एक आर्किटेक्ट यह सुनिश्चित करता है कि एक ऐप का कार्यान्वयन इसे अपने आसपास के घटकों के साथ एक स्केलेबल, रखरखाव योग्य तरीके से बातचीत करने की अनुमति देता है। आसपास के घटकों में डेटाबेस, लॉगिंग इन्फ्रास्ट्रक्चर और अन्य माइक्रोसर्विसेज शामिल हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/application-architect.md:18]: कार्यान्वयन means execution. Don't we mean to say execution environment in this?

**Review by divya-mohan0209 — COMMENTED** [review]
Left some suggestions, hope you find them helpful!

File: content/hi/docs/reference/glossary/application-architect.md (hunk 1 above)
Comment (bishal7679) [reply content/hi/docs/reference/glossary/application-architect.md:7]: Yes Actually I was thinking to use रचना but anyone might get confused with design
so I did not use it
Nice suggestion 👍 


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Comment by divya-mohan0209** [comment]
@bishal7679 : PTAL at this PR whenever free.

**Comment by divya-mohan0209** [comment]
@bishal7679 I'd recommend अमल since it seems simpler.

Also, you'll need to sync your fork and pull the latest branch changes for ALL your PRs

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by divya-mohan0209** [comment]
@Babapool @verma-kunal @Garima-Negi @anubha-v-ardhan : PTAL

**Review by Babapool — COMMENTED** [review]
/lgtm

**Review by divya-mohan0209 — COMMENTED** [review]
Taking approval from comment by @Babapool and approving the PR.
/approve

**Pull request merged**